setup.py

```diff
@@ -1193,174 +1193,19 @@ def main():
-        "include/torch/csrc/distributed/rpc/*.h",
-        "include/torch/csrc/distributed/autograd/context/*.h",
-        "include/torch/csrc/distributed/autograd/functions/*.h",
-        "include/torch/csrc/distributed/autograd/rpc_messages/*.h",
-        "include/torch/csrc/dynamo/*.h",
-        "include/torch/csrc/inductor/*.h",
-        "include/torch/csrc/inductor/aoti_include/*.h",
-        "include/torch/csrc/inductor/aoti_package/*.h",
-        "include/torch/csrc/inductor/aoti_runner/*.h",
-        "include/torch/csrc/inductor/aoti_runtime/*.h",
-        "include/torch/csrc/inductor/aoti_torch/*.h",
-        "include/torch/csrc/inductor/aoti_torch/c/*.h",
-        "include/torch/csrc/inductor/aoti_torch/generated/*.h",
-        "include/torch/csrc/inductor/aoti_torch/generated/extend/*.h",
-        "include/torch/csrc/inductor/cpp_wrapper/*.h",
-        "include/torch/csrc/inductor/cpp_wrapper/device_internal/*.h",
-        "include/torch/csrc/jit/*.h",
-        "include/torch/csrc/jit/backends/*.h",
-        "include/torch/csrc/jit/generated/*.h",
-        "include/torch/csrc/jit/passes/*.h",
-        "include/torch/csrc/jit/passes/quantization/*.h",
-        "include/torch/csrc/jit/passes/utils/*.h",
-        "include/torch/csrc/jit/runtime/*.h",
-        "include/torch/csrc/jit/ir/*.h",
-        "include/torch/csrc/jit/frontend/*.h",
-        "include/torch/csrc/jit/api/*.h",
-        "include/torch/csrc/jit/serialization/*.h",
-        "include/torch/csrc/jit/python/*.h",
-        "include/torch/csrc/jit/mobile/*.h",
-        "include/torch/csrc/jit/testing/*.h",
-        "include/torch/csrc/jit/tensorexpr/*.h",
-        "include/torch/csrc/jit/tensorexpr/operators/*.h",
-        "include/torch/csrc/jit/codegen/cuda/*.h",
-        "include/torch/csrc/onnx/*.h",
-        "include/torch/csrc/profiler/*.h",
-        "include/torch/csrc/profiler/orchestration/*.h",
-        "include/torch/csrc/profiler/standalone/*.h",
-        "include/torch/csrc/profiler/stubs/*.h",
-        "include/torch/csrc/profiler/unwind/*.h",
-        "include/torch/csrc/profiler/python/*.h",
-        "include/torch/csrc/utils/*.h",
-        "include/torch/csrc/tensor/*.h",
-        "include/torch/csrc/lazy/backend/*.h",
-        "include/torch/csrc/lazy/core/*.h",
-        "include/torch/csrc/lazy/core/internal_ops/*.h",
-        "include/torch/csrc/lazy/core/ops/*.h",
-        "include/torch/csrc/lazy/python/python_util.h",
-        "include/torch/csrc/lazy/ts_backend/*.h",
-        "include/torch/csrc/xpu/*.h",
-        "include/pybind11/*.h",
-        "include/pybind11/detail/*.h",
-        "include/pybind11/eigen/*.h",
-        "include/TH/*.h*",
-        "include/TH/generic/*.h*",
-        "include/THC/*.cuh",
-        "include/THC/*.h*",
-        "include/THC/generic/*.h",
-        "include/THH/*.cuh",
-        "include/THH/*.h*",
-        "include/THH/generic/*.h",
-        "include/sleef.h",
+        "include/**/*.h",
```

Comment:
Let me know if these includes should be more restrictive

Reply:
Is there a way to block-list say anything that starts with `internal`?

Reply:
No, but we can just make it more specific to like how it was originally, but make it include all subfolders

Reply:
@malfet You mean starts with internal right? Not has internal anywhere in the path?

Reply:
I mean we could manually filter it ouf the list with pure Python, i suppose?

Reply:
@malfet Is there a list of top level folder includes you want to block?